gpflow/utilities/utilities.py

```diff
@@ -15,6 +15,7 @@
     "multiple_assign",
     "training_loop",
     "print_summary",
+    "leaf_components"
```

Comment:
```suggestion
    "leaf_components",
```
to make future diffs nicer ;)